.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,9 @@
+- id: trivy
+  name: trivy
+  entry: trivy
+  language: golang
+- id: trivyfs
+  name: trivyfs
+  entry: trivy fs --exit-code 1 .
```

Comment:
`Trivy` repository has many test files with vulnerabilities.
Scan result will always return exit code 1 for these files.
I think we can only select 1 file.

Reply:
Sorry for misleading you.
Yesterday I made a mistake with the logic of using `pre-commit`.
Thanks for help me understand.